app/Console/Commands/Ghostbuster.php

```diff
@@ -93,9 +93,14 @@ public function handle()
 
 				// for normal pictures
 				$to_delete[] = 'small/' . $url;
-				$to_delete[] = 'small/' . $photoName[0] . '@2x.' . $photoName[1];
 				$to_delete[] = 'medium/' . $url;
-				$to_delete[] = 'medium/' . $photoName[0] . '@2x.' . $photoName[1];
+				if (count($photoName) === 2) {
+					$to_delete[] = 'small/' . $photoName[0] . '@2x.' . $photoName[1];
```

Comment:
why no use of the Helper here ? O.o

Reply:
Good point. I guess I got blindsided by the existing code...